frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -263,7 +264,7 @@ describe("scenarios > visualizations > pivot tables", () => {
             .then($value => {
               cy.visit($value);
             });
-          cy.findByText(test.subject);
+          cy.findAllByText(test.subject); // the inspector only saw one, but findByText failed due to multiple elements
```

Comment:
This flake was a mystery to me. I went back to the DOM snapshot in the Cypress UI and `contains` only found "pivot table dashboard" once, but `findByText` had failed because there were multiple elements.

Reply:
😳

Reply:
😳